tasks/base_tasks.py

```diff
@@ -1592,3 +1610,133 @@ def collect_meta_wrappers(test_module=None, test_all=True):
             yield t, params
             if not test_all:
                 break
+
+
+class CreateRepoTable(Task):
+    schema = 'repo'
+    table = 'filerepo'
+
+    def run(self):
+        session = current_session()
+
+        create_schema = '''
+                        CREATE SCHEMA IF NOT EXISTS "{schema}"
+                        '''.format(schema=self.output().schema)
+        session.execute(create_schema)
+
+        create_table = '''
+                        CREATE TABLE IF NOT EXISTS "{schema}".{table} (
```

Comment:
Minor: I'd quote consistently, either both schema and table or none.

Reply:
Quoting is dangerous. The schema name needs to be quoted because the schemas almost always include a dot (`es.ine`), and thus in all the DO functions and ETL tasks we use quotes for it. But we never use quotes for table names and if we quote it here we would need to modify all the functions and tasks to add quotes for table names (a table created `create table myschema."MyTable"` will return an error `table doesn't exist` if accessed without quotes `select * from myschema.MyTable`)

Reply:
Oh, I was just plain wrong about this, sorry! We've had a lot of problems with table naming in Builder and I recalled ending up quoting pretty much everything.